pages/_app.js

```diff
@@ -1,10 +1,14 @@
 import App from 'next/app';
+import TagManager from 'react-gtm-module';
 import Layout from 'components/Layout';
 import './stylesheets/all.scss';
 
 export default class MyApp extends App {
   componentDidMount = () => {
     window.GOVUKFrontend.initAll();
+    TagManager.initialize({
+      gtmId: process.env.NEXT_PUBLIC_GTM_ID
```

Comment:
Does it need to be removed from .env.sample?

Reply:
we're still using it